examples/Feeds/RemoveFlightsFeedItemAttributeValue.cs

```diff
@@ -0,0 +1,307 @@
+            int attributeIndex = GetAttributeIndex(feedItem, feedItemAttributeValue);
+
+            if (attributeIndex == -1)
+            {
+                throw new ArgumentException("No matching feed attribute found for " +
+                                            $"value '{feedItemAttributeValue}'.");
+            }
+
+            // Returns the feed item with the removed FeedItemAttributeValue. Any
+            // FeedItemAttributeValues that are not included in the updated FeedItem will be removed
+            // from the FeedItem, which is why you must create the FeedItem from the existing
+            // FeedItem and set the field(s) that will be removed.
+            feedItem.AttributeValues.RemoveAt(attributeIndex);
+            return feedItem;
+        }
+
+        /// <summary>
+        /// Retrieves a feed item and its attribute values given a resource name.
+        /// </summary>
+        /// <param name="client">The Google Ads API client.</param>
+        /// <param name="customerId">The client customer ID.</param>
+        /// <param name="feedItemResourceName">The feed item resource name.</param>
+        /// <returns>FeedItem with the given resource name.</returns>
+        private FeedItem GetFeedItem(GoogleAdsClient client, in long customerId,
+            string feedItemResourceName)
+        {
+            // Gets the Google Ads service.
+            GoogleAdsServiceClient googleAdsServiceClient = client.GetService(
+                Services.V4.GoogleAdsService);
+
+            // Constructs the query.
+            string query = $@"SELECT feed_item.attribute_values
+                            FROM feed_item
+                            WHERE feed_item.resource_name = '{feedItemResourceName}'";
+
+
+            // Issues the search request.
+            PagedEnumerable<SearchGoogleAdsResponse, GoogleAdsRow> searchResponse =
+                googleAdsServiceClient.Search(customerId.ToString(), query);
+
+            // Returns the feed item attribute values.
+            return searchResponse.First().FeedItem;
+        }
+
+
+        /// <summary>
+        /// Gets the index of the attribute. This is needed to specify which FeedItemAttributeValue
+        /// will be removed in the given FeedItem.
+        /// </summary>
+        /// <param name="feedItem">The FeedItem that will be updated.</param>
+        /// <param name="removeFeedItemAttributeValue">The new FeedItemAttributeValue that will be
+        ///     removed.</param>
+        /// <returns>The integer index of the attribute.</returns>
+        private int GetAttributeIndex(FeedItem feedItem,
+            FeedItemAttributeValue removeFeedItemAttributeValue)
+        {
+            int attributeIndex = -1;
+
+            // Loops through attribute values to find the index of the FeedItemAttributeValue to
+            // update.
+            foreach (FeedItemAttributeValue feedItemAttributeValue in feedItem.AttributeValues)
```

Comment:
We could possibly use Linq here. See the accepted answer at https://stackoverflow.com/questions/9300169/linq-indexof-a-particular-entry for some inspiration.

Reply:
Had the same thought but figured it better for clarity to leave as a loop in a separate method.  Updated and moved to a one-line